api/organisations/invites/views.py

```diff
@@ -59,6 +59,12 @@ def join_organisation_from_link(request, hash):
 
     invite = get_object_or_404(InviteLink, hash=hash)
 
+    organisation = Organisation.objects.get(id=invite.organisation.id)
+    if organisation.over_plan_seats_limit(shift=1):
+        raise PermissionDenied(
+            "The seats limit has been reached. Please contact your organisation."
+        )
```

Comment:
Should reuse the logic [here](https://github.com/Flagsmith/flagsmith/blob/main/api/organisations/invites/views.py#L131-L136)

Reply:
done